common/src/main/java/com/example/demo/ShoeApiEndpoints.java

```diff
@@ -0,0 +1,22 @@
+package com.example.demo;
+
+/**
+ * <p>
+ * <p>
+ * Centralise l'ensemble des endpoints disponibles pour les apis RESTS.
```

Comment:
I would suggest being careful with casing.
_API_ is an acronym, so is _REST_, hence it has no plural, and _RESTS_ is not an acronym.